SickDesignSystem/sds
# 22 doc add pages with static css core tokens to storybook

# Pull Request

## 📖 Description

This PR adds the first meaningful content to our Storybook instance:

1. Custom start page (Content taken from original design docs)
2. Removed storybook demo stories
3. (Semi-) Static color overview (with text taken from original Design Docs)
4. Dynamic design tokens for colors
5. Readme and Changelog file includes for the `@sick-design-system/design-tokens` package.

### 🎫 Issues

Closes #22.

## 👩‍💻 Reviewer Notes

I have played around with different documentation formats and have created a baseline structure. Would be interested to hear what you think about the structure. Especially the color section was split into different subpages as it was quite long.

Also I have opted to not add any non color Design-Tokens yet. The cause of this are the proposed changes in #36. It will take some time to make sure our token structure is completely settled before using the raw values all over the library.

## 📑 Test Plan

- Issue `pnpm i -r` in the repository root
- Make sure `pnpm lint` runs through fine
- Check the local version of storybook via `cd packages/docs && pnpm start`
- Check the static version of storybook via `cd packages/docs && pnpm build`. Are all assets loading when loading as static version?

## ✅ Checklist

### General
